pkgs/development/interpreters/python/hooks/fix-python-deps.sh

```diff
@@ -0,0 +1,76 @@
+# shellcheck shell=bash
+
+# Setup hook that fixes Python dependencies versions.
+#
+# Example usage in a derivation:
+#
+#   { …, pythonPackages, … }:
+#
+#   pythonPackages.buildPythonPackage {
+#     …
+#     nativeBuildInputs = [ pythonPackages.fixPythonDepsHook ];
+#
+#     # Already the default, only change if necessary
+#     requirementsTxt = "requirements.txt";
+#
+#     # This will remove the upperbound version of the package
+#     # e.g.: abc>1<=2 -> abc>1
```

Comment:
```suggestion
#     # e.g.: abc>1,<=2 -> abc>1
```
Isn't that the normal syntax?
How about ``==``, ``!=``, ``<`` and ``>``?

Reply:
> Isn't that the normal syntax?
Fixed.
> How about ==, !=, < and >?
Switched to a proper `requirements.txt`. Shouldn't be an issue anymore.